src/main/java/com/ita/if103java/ims/dao/impl/UserDaoImpl.java

```diff
@@ -286,8 +293,9 @@ private PreparedStatement getPreparedStatement(User user, Connection connection)
         public static final String SQL_SELECT_ALL_USERS = """
```

Comment:
Do not know if we need this query. 
Even if we need it it is not filtering by the account id

Reply:
I agree, but for now, probably let it be?

Reply:
Then do not spent time on what is not going to be used